docs/user-guide/deploying-application/triggering-cd.md

```diff
@@ -10,9 +8,59 @@ Select an image to deploy and then click on `Deploy` to trigger the CD pipeline.
 
 ![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/deploying-application/triggering-cd/deploy.jpg)
 
-The current deployed images are tagged as `Deployed on <Environment name>`.
+The currently deployed images are tagged as `Active on <Environment name>`.
+
+## Manual Approval for Deployment
+
+When manual approval is enabled for the deployment pipeline, you are required to request image approval before each deployment. Alternatively, you can only deploy images that have already been approved once.
+
+When no approved image is available or the deployed image is already deployed, you will not see any image available to be deployed when you click on the `Select Image` button.
+
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/deploying-application/triggering-cd/no-approved-image.jpg)
+
+### Request For Image Approval
+
+To request for image approval, please follow these steps:
+
+1. Navigate to `Build & Deploy` page, and click on the `Approval for deployment` button.
+
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/deploying-application/triggering-cd/deployment-approval-button.jpg)
+
+2. Click on `Request Approval` button present on the image for which you want to request approval and the click on `Submit Request`.
+
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/deploying-application/triggering-cd/request-approval.jpg)
+
+**NOTE:** You have the option to cancel the approval request from the `Approval Pending` section.
+
+### Approve Image Approval Request
+
+To approve an image approval request, please follow these steps:
+
+1. Go to the `Build & Deploy` page and click on the `Approval for deployment` button.
+
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/deploying-application/triggering-cd/deployment-approval-button.jpg)
+
+2. Switch to the `Approval Pending` tab. Here, you will find all the images that are awaiting approval.
+
+3. Click on the `Approve` button and the click on `Approve Request` to grant the approval for the request.
```

Comment:
Click the `Approve` button, and then `Approve Request` to grant approval to the request.